shell/platform/embedder/tests/embedder_config_builder.h

```diff
@@ -31,13 +31,15 @@ using UniqueEngine = fml::UniqueObject<FlutterEngine, UniqueEngineTraits>;
 class EmbedderConfigBuilder {
  public:
   enum class InitializationPreference {
-    kInitialize,
+    kSnapshotsInitialize,
```

Comment:
Can you not just initialize snapshots in JIT and ELF in AOT? This seems like an unnecessary addition to the test harness API. There is no configuration in which both are valid inputs for the fixture.